packages/components/src/Combobox/components/Content/Content.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import classnames from "classnames";
+import styles from "./Content.css";
+import { Icon } from "../../../Icon";
+import { ComboboxContext } from "../../ComboboxProvider";
+
+interface ContentProps {
+  /**
+   * options for the combobox
+   */
+  readonly options: string[];
+
+  /**
+   * onSelection handler.
+   */
+  onSelection: (selection: string) => void;
+
+  /**
+   * actions
+   */
+  readonly children: React.ReactNode;
+}
+
+export function Content(props: ContentProps): JSX.Element {
+  const [searchValue, setSearchValue] = useState<string>("");
+  const [selectedOption, setSelectedOption] = useState<string>("");
```

Comment:
why is this a state? Should the consumer decide which one is selected?

Reply:
good question

not as of right now but coming soon in another subtask, yes!

imagine using a combobox for something in the edit state. you'd already have a selection, so we need to support the consumer being able to pre-set what is selected via `setSelectedOption`

Reply:
oh good. as long as there's subtask and we don't forget about it!